functions.php

```diff
@@ -1319,3 +1340,287 @@ function wp_autoupdates_themes_bulk_actions_handle( $redirect_to, $doaction, $it
 	}
 }
 add_action( 'handle_network_bulk_actions-themes-network', 'wp_autoupdates_themes_bulk_actions_handle', 10, 3 );
+
+/**
+ * Disable auto updates via Ajax.
+ */
+function wp_autoupdates_disable_auto_updates() {
+	$nonce = sanitize_text_field( $_POST['nonce'] );
+	$type  = sanitize_text_field( $_POST['type'] );
+	$asset = sanitize_text_field( urldecode( $_POST['asset'] ) );
+	if ( ! wp_verify_nonce(
```

Comment:
For consistency with other WordPress Ajax functions, like `wp_ajax_update_plugin` it might be worth using `check_ajax_referer` instead.

Reply:
@TimothyBJacobs Changed with https://github.com/WordPress/wp-autoupdates/pull/61/commits/bbc30fd05cba1abdf0b3980ca8b50fe658cecbe8